homebrewing/src/components/Finish/Finish.jsx

```diff
@@ -59,7 +59,7 @@ export default function Finish() {
     } else {
       return
     }
-  }, [])
+  }, [coffee, grind, menuName, methodName, roasting, userEmail, water])
```

Comment:
Have you tested that this still works as expected or have you just added these dependencies because eslint suggested them?

Reply:
Sorry Eryk I put all those dependencies just beacuase eslint showed me.
But I tested  that component and seems ok at least from user's prespective.